### PR TITLE
Added values of WindowStartupLocation enum

### PR DESCRIPTION
**Pull request by yeremee**
# Title

Added values of WindowStartupLocation enum

### CODE REVIEW

File: docs/framework/wpf/app-development/wpf-windows-overview.md [hunk 1]
```diff
@@ -302,11 +302,11 @@ Users interact with              [!INCLUDE[TLA#tla_wpf](../../../../includes/tla
...
--   <xref:System.Windows.WindowStartupLocation> (default)  
+-   <xref:System.Windows.WindowStartupLocation.CenterOwner> (default)  
   
--   <xref:System.Windows.WindowStartupLocation>  
+-   <xref:System.Windows.WindowStartupLocation.CenterScreen>  
   
--   <xref:System.Windows.WindowStartupLocation>  
+-   <xref:System.Windows.WindowStartupLocation.Manual>  
   
  If the startup location is specified as                  <xref:System.Windows.WindowStartupLocation>, and the                  <xref:System.Windows.Window.Left%2A> and                  <xref:System.Windows.Window.Top%2A> properties have not been set,                  <xref:System.Windows.Window> will ask                  [!INCLUDE[TLA2#tla_mswin](../../../../includes/tla2sharptla-mswin-md.md)] for a location to appear in.
```
Comment (rpetrusha) [inline docs/framework/wpf/app-development/wpf-windows-overview.md]: @yeremee, would you mind also changing this to <xref:System.WIndows.WindowStartupLocation.Manual?displayProperty=fullName>?

**Review by rpetrusha — CHANGES_REQUESTED** [review]
@yeremee, thanks for fixing this! Could you also make one more change?

**Comment by mairaw** [comment]
@rpetrusha how about we merge this one and continue the changes on a separate PR? there are more links to enum fields missed in the conversion.

**Comment by rpetrusha** [comment]
@mairaw, yes, I agree.

**Review by rpetrusha — APPROVED** [review]

File: docs/framework/wpf/app-development/wpf-windows-overview.md (hunk 1 above)
Comment (yeremee) [inline docs/framework/wpf/app-development/wpf-windows-overview.md]: Ok.

**Pull request merged**